tests/foreman/destructive/test_puppetplugin.py

```diff
@@ -133,7 +133,7 @@ def test_positive_enable_disable_logic(target_sat, capsule_configured):
     assert_puppet_status(target_sat, expected=False)
 
 
-@pytest.mark.rhel_ver_match('[^9]')
+@pytest.mark.rhel_ver_match('[7,8]')
```

Comment:
Any reason to exclude el9 here?
```suggestion
@pytest.mark.rhel_ver_match('[^6]')
```

Reply:
The reason is that rhel9 is not supported by puppetlabs yet.